WcaOnRails/app/models/competition_big_json.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class CompetitionBigJson < ApplicationRecord
+  belongs_to :competition, required: true
```

Comment:
Weird! I didn't expect that change! But my point was that it seems natural to have it optional. Many competitions won't have any CCIF data, and that seems fine.